src/files/XmlFile.ts

```diff
@@ -129,7 +136,11 @@ export class XmlFile {
      */
     public getAvailableScriptImports() {
         return this.cache.getOrAdd('allAvailableScriptImports', () => {
-            let allDependencies = this.getAllScriptImports();
+
+            let allDependencies = this.getAllDependencies()
+                //skip typedef files
+                .filter(x => extname(x) !== '.d.bs');
```

Comment:
Should they be replaced by `.brs` files?

Reply:
No, the .bs/.brs files are already there. For every .bs/.brs file, the XmlFile _also_ adds the corresponding `.d.bs` file during initialization (see [this](https://github.com/rokucommunity/brighterscript/pull/188/files#diff-c2462ef143ff214d1683501300a6cc2960cf54d7ba36e7606f7383c95021a240R430) new block of code)